libs/langchain/langchain/chains/api/base.py

```diff
@@ -198,7 +198,7 @@ async def acall_model(state: ChainState, config: RunnableConfig):
         api_docs: str
         question_key: str = "question"  #: :meta private:
         output_key: str = "output"  #: :meta private:
-        limit_to_domains: Optional[Sequence[str]]
+        limit_to_domains: Optional[Sequence[str]] = Field(default_factory=list)
```

Comment:
@baskaryan -- this is why I was afraid of doing it automatically -- it completely changes the semantics in this case and would introduce a CVE